polynote-frontend/polynote/main.ts

```diff
@@ -28,7 +28,7 @@ SocketSession.global;
 const mainUI = new MainUI();
 const mainEl = document.getElementById('Main');
 mainEl?.appendChild(mainUI.el);
-const path = unescape(window.location.pathname.replace(new URL(document.baseURI).pathname, ''));
+const path = unescape(decodeURIComponent(window.location.pathname.replace(new URL(document.baseURI).pathname, '')));
```

Comment:
Is the `unescape` still necessary? I hadn't heard of `decodeURIComponent` but it looks like it does everything `unescape` was going to do as well.

Reply:
Sorry, I'm unfamiliar with js and almost made a mistake.
refer to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape
This place no longer needs `unescape`. I remote it  and test again, it seems that there is no problem at present.

Reply:
Awesome, LGTM! Thanks @akiyamaneko!